docs/tutorial.md

```diff
@@ -200,24 +176,35 @@ There are a couple of ways to create the CRD. You can either create the file
 manually. Or let the quarkus extensions defined in `pom.xml` use the annotations
 on your Spec/Status classes to create the crd files for you.
 
-#### Manually create `crd.yaml`
+#### Via Quarkus extension
+
+Running `mvn clean install` will invoke the CRD generator extension which will analyze
+the annotations on the model objects,  `Memcached`, `MemcachedSpec`,
+`MemcachedStatus`, and generate the CRD in `target/kubernetes`.
+
+CRD generated in `memcacheds.cache.example.com-v1.yml`.
 
-Create a file with the name `crd.yaml`. A CRD enables users to add their
-own/custom objects to the Kubernetes cluster. Below you will find an example
-`Memcached` CRD.
+```
+.
+├── kubernetes.json
+├── kubernetes.yml
+└── memcacheds.cache.example.com-v1.yml
+
+0 directories, 3 files
+```
+
+The content of the `memcacheds.cache.example.com-v1.yml` file is as shown below.
 
 ```
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
```

Comment:
I'm not sure how you generated this CRD but it doesn't look like it got generated properly. Perhaps it got generated before adding fields to `MemcachedSpec` and `MemcachedStatus` classes?

Reply:
Oh, yes! absolutely. I ran `mvn clean install` just after `create api` command. I will update the code and re-generate it thru compilation.